From 75b02abb0f8c4826c0bf435422b5d659057f664a Mon Sep 17 00:00:00 2001
From: Mario Ceresa <mrceresa@gmail.com>
Date: Wed, 10 Oct 2012 17:39:14 +0200
Subject: [PATCH 09/23] Bumped up version to 1.14

Signed-off-by: Mario Ceresa <mrceresa@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 71f48bc..c532b01 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,7 @@ ENDIF( COMMAND CMAKE_POLICY )
 PROJECT(vxl)
 
 SET(VXL_MAJOR_VERSION 1)
-SET(VXL_MINOR_VERSION 13)
+SET(VXL_MINOR_VERSION 14)
 SET(VXL_BUILD_VERSION 0)
 SET(VXL_VERSION
   "${VXL_MAJOR_VERSION}.${VXL_MINOR_VERSION}.${VXL_BUILD_VERSION}")
-- 
1.7.11.7

