From 2721eacfd99e884904adbef9b1fe77530bcbbe4c Mon Sep 17 00:00:00 2001
From: Mario Ceresa <mrceresa@gmail.com>
Date: Wed, 10 Oct 2012 17:39:50 +0200
Subject: [PATCH 11/23] Do not use bundled minizip

Signed-off-by: Mario Ceresa <mrceresa@gmail.com>
---
 contrib/brl/b3p/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/contrib/brl/b3p/CMakeLists.txt b/contrib/brl/b3p/CMakeLists.txt
index 7e61ffb..a0cc98e 100644
--- a/contrib/brl/b3p/CMakeLists.txt
+++ b/contrib/brl/b3p/CMakeLists.txt
@@ -16,10 +16,10 @@ MARK_AS_ADVANCED( VXL_FORCE_B3P_EXPAT )
 SUBDIRS(expat)
 SUBDIRS(expatpp)
 
-OPTION( VXL_FORCE_B3P_SHAPELIB "Use B3P instead of any native SHAPELIB library?" YES)
+OPTION( VXL_FORCE_B3P_SHAPELIB "Use B3P instead of any native SHAPELIB library?" NO)
 MARK_AS_ADVANCED( VXL_FORCE_B3P_SHAPELIB )
 SUBDIRS(shapelib)
 
-SUBDIRS(minizip)
+#SUBDIRS(minizip)
 
 
-- 
1.7.11.7

