From aefaa1d2e0e621d7f3ef2ea420c92bd0c0f29e64 Mon Sep 17 00:00:00 2001
From: Mario Ceresa <mrceresa@gmail.com>
Date: Fri, 2 Nov 2012 14:40:49 +0100
Subject: [PATCH 19/23] Removed box2m which requires OpenCL libraries

---
 contrib/brl/bseg/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/contrib/brl/bseg/CMakeLists.txt b/contrib/brl/bseg/CMakeLists.txt
index b0e3a56..31ea0b5 100644
--- a/contrib/brl/bseg/CMakeLists.txt
+++ b/contrib/brl/bseg/CMakeLists.txt
@@ -48,10 +48,10 @@ SUBDIRS(boct)
 SUBDIRS(boxm)
 
 # updated, gpu compatible octree based modeling
-SUBDIRS(boxm2)
+#SUBDIRS(boxm2)
 
 # updated, gpu compatible octree based modeling
-SUBDIRS(boxm2_batch)
+#SUBDIRS(boxm2_batch)
 
 # octree based modeling python interface
 SUBDIRS(boxm_batch)
@@ -72,4 +72,4 @@ SUBDIRS(bapl)
 SUBDIRS(boxm_bridge)
 
 #multi gpu library
-SUBDIRS(boxm2_multi)
+#SUBDIRS(boxm2_multi)
-- 
1.7.11.7

